src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -214,6 +215,7 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
     <Compile Include="System\Net\Http\HttpClientHandler.Core.cs" />
+    <Compile Include="System\Net\Http\HttpClientHandler.Win32.cs" />
```

Comment:
It looks weird to be including Win32.cs file for Unix...

Reply:
Ok I'm going to just go with .NetCoreApp.cs for this. We already have .Unix, .Windows, .Core.

Reply:
Nit: It should be all lowercase netcoreapp.cs to match the convention